test/recipes/80-test_cmp_http_data/Mock/setup-mock.sh

```diff
@@ -0,0 +1,159 @@
+#! /bin/bash
+set -e
+#set -x  #for debug
+
+# This script generates the certificates needed for the CMP server and the signer
+
+if [[ ! -f $mkcert_sh ]]; then
+    mkcert_sh="../../../certs/mkcert.sh"
+fi
+
+# sever certificate algorithms
+if [ -z "$server_rootca_keyalg" ]; then
+    server_rootca_keyalg="MLDSA65"
+fi
+if [ -z "$server_leaf_keyalg" ]; then
+    server_leaf_keyalg="SLH-DSA-SHAKE-192s"
+fi
+# rootCACert update test case
+if [ -z "$new_rootca_keyalg" ]; then
+    new_rootca_keyalg="SLH-DSA-SHAKE-192s"
+fi
+
+# End-entity certificate algorithms
+if [ -z "$signer_rootca_keyalg" ]; then
+    signer_rootca_keyalg="MLDSA65"
+fi
+if [ -z "$signer_interca_keyalg" ]; then
+    signer_interca_keyalg="MLDSA65"
+fi
+if [ -z "$signer_subinterca_keyalg" ]; then
+    signer_subinterca_keyalg="MLDSA65"
+fi
+if [ -z "$signer_leaf_keyalg" ]; then
+    signer_leaf_keyalg="SLH-DSA-SHAKE-192s"
+fi
+
+# CMP server certificate
+rename_serverfiles() {
+    echo "Renaming server files"
+    rm server_root-key.pem server_root-pubkey.pem newWithNew-key.pem newWithNew-pubkey.pem
```

Comment:
Better remove the intermediate files server_root-pubkey.pem and newWithNew-pubkey.pem locally, right after they were used.